.env.example

```diff
@@ -0,0 +1,3 @@
+# VITE_API_PATH=http://127.0.0.1:8000
+
+VITE_API_PATH=https://magic-box.hub.lucien.ink:2016
```

Comment:
Can you hide this from both file and commit history?

Reply:
done with a squash rebase, you could just remove this review comment